sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/derived/SaplingReceivedNoteTable.kt

```diff
@@ -5,7 +5,7 @@ import cash.z.ecc.android.sdk.internal.db.queryAndMap
 import cash.z.ecc.android.sdk.model.ZcashNetwork
 import java.util.Locale
 
-internal class ReceivedNoteTable(
+internal class SaplingReceivedNoteTable(
```

Comment:
Does it still make sense to use the `sapling_received_notes` table directly? The `tx_outputs` view is potentially a better source of this data.

Reply:
Yes, it is but the changes are not small so that can be adopted later as long as the SDK is cherrypicking data from them as it is currently now.